features/steps/step_checking_status.py

```diff
@@ -0,0 +1,61 @@
+from behave import when, then
+from selenium import webdriver
+from selenium.webdriver.support.wait import WebDriverWait
+from helpers.icon_in_div_changed import IconInDivChanged
+from selenium.webdriver.support import expected_conditions as ec
+from selenium.webdriver.common.by import By
+
+
+FULL_NAME = None
+driver = webdriver.Chrome
+status_shortcuts = {'available': 'icon-avail',
+                    'away': 'icon-xa',
+                    'do not disturb': 'icon-dnd'}
+
```

Comment:
In feature you need to remove logical code from steps files, if it is possible.

Reply:
Done. Thank you.